services/movie.py

```diff
@@ -0,0 +1,42 @@
+from django.db.models import QuerySet
+from db.models import Genre, Actor, Movie
+
+
+def get_movies(genres_ids: list[Genre] = None, /,
+               actors_ids: list[Actor] = None
+               ) -> QuerySet:
+    query = Movie.objects.all()
+    if genres_ids and actors_ids:
+        query = Movie.objects.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
+    elif genres_ids:
+        query = Movie.objects.filter(genres__id__in=genres_ids)
+    elif actors_ids:
+        query = Movie.objects.filter(actors__id__in__=actors_ids)
```

Comment:
There is a typo in the filter condition for `actors_ids`. It should be `actors__id__in=actors_ids` instead of `actors__id__in__=actors_ids`. This is an error related to the task requirements, as it will cause a syntax error.